### ISSUE TITLE
Create login page for the application

### ISSUE THREAD

**Issue by ankitshaw** [CodingIndia/travel-safe#9]
You need to create a login page for the application.

**Comment by akshaydhiran** [comment]
I want to work on this issue.

**Comment by vineetjk** [comment]
I want to work on this

**Comment by ankitshaw** [comment]
@vineetjk issue assigned

**Comment by ankitshaw** [comment]
@akshaysingh5 you have been assigned another issue. For everyones benefit we are assigning one issue per person at a time till you PR is accepted. Thank You.

**Comment by FrozenGirl-spd** [comment]
Can i work on this?

**Comment by neversettle17-101** [comment]
I would like to work on this

**Comment by SnegaRamesh** [comment]
I would like to work on this issue.Can u please assign?

**Comment by shwhite17** [comment]
assign it to me please ?

**Comment by ankitshaw** [comment]
@vineetjk  are you still working on the issue?

**Comment by vineetjk** [comment]
yes i have completed this issue
but need to push the code

**Comment by FrozenGirl-spd** [comment]
Can i work on the issue?
@ankitshaw

**Comment by ankitshaw** [comment]
Hi @FrozenGirl-spd, assigned the issue to you!.

**Comment by ankitshaw** [comment]
Hi @FrozenGirl-spd  are you working on the issue?

**Comment by tanmayrauth** [comment]
@FrozenGirl-spd Please do let us know the status otherwise we will have to assign it to someone else

**Comment by Rukmini-Meda** [comment]
I would like to work on this issue. Please assign this to me as part of GSSOC'20.

**Comment by tanmayrauth** [comment]
assigned to @Rukmini-Meda 
Once the UI part is done then we can focus over backend in different issue.
